### PR TITLE
Add FBT for replication controller details

### LINKED ISSUE #503: Finish remaining FBT sheets
#### Issue details
We should complete the remaining FBT sheets.

Missing:
- Replication Details
- Logs

Any comments on the process? Should we adapt or change something?

@floreks , @digitalfishpond @maciaszczykm @f-higashi 
please pick

### PR DESCRIPTION
**Pull request by maciaszczykm**
Fixes #503 issue.

---
This change is [Review on Reviewable](https://reviewable.io/reviews/kubernetes/dashboard/512)

### CODE REVIEW

**Comment by maciaszczykm** [comment]
@cheld Could you take a look?

File: docs/test/replication-controller-details.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+* sidebar is available
+* application details are valid and matching data from `kubectl` command
+* all application actions are available
+* warning events have warning icon nearby to their messages
+
+## Specific factors
+
+| Factor                        | 1                               | 2                       | 3              | 4             | 5            | 6               | 7          | 8      | Comment                                                                               |
+|-------------------------------|---------------------------------|-------------------------|----------------|---------------|--------------|-----------------|------------|--------|---------------------------------------------------------------------------------------|
+| Screen size                   | normal                          | mobile                  | narrow         | middle size   | wide         | active resizing |            |        |                                                                                       |
```
Comment (cheld) [inline docs/test/replication-controller-details.md]: Gain, I would suggest to drop and leave only the general factor screen size

File: docs/test/replication-controller-details.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+| Pods count                    | 1                               | odd                     | even           | 20-30         |              |                 |            |        |                                                                                       |
```
Comment (cheld) [inline docs/test/replication-controller-details.md]: 1 | 1-10 | > 100

File: docs/test/replication-controller-details.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+| Events count                  | 0                               | 0-10                    | more than 10   |               |              |                 |            |        | events number can be changed for example by scaling replication controller pods count |
```
Comment (cheld) [inline docs/test/replication-controller-details.md]: 0 (thats a good case!! but how to test? ) 
0-10
>1000  (the number of events increases with the number of pods)

Is there an upper limit of events? How many events are likely to occur for each pod?

File: docs/test/replication-controller-details.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+| Replication Controller status | valid state                     | invalid error state     |                |               |              |                 |            |        |                                                                                       |
```
Comment (cheld) [inline docs/test/replication-controller-details.md]: The card status is not relevant - I think. The status of each single event is marked. There are a lot of bugs around highlighting events. Please investigate and add factors to test.

File: docs/test/replication-controller-details.md (hunk 1 above)
Comment (maciaszczykm) [inline docs/test/replication-controller-details.md]: In this case it's even more important than on replication controller list page. We have three page breaks here. One with full view, second with sidebar moved into tab and last one with responsive table. I'd leave it like it's done at the moment.

File: docs/test/replication-controller-details.md (hunk 3 above)
Comment (maciaszczykm) [inline docs/test/replication-controller-details.md]: > 0 (thats a good case!! but how to test? ) 

With time some of events disappear, but I don't know if it's a rule...

Is there an upper limit of events? How many events are likely to occur for each pod?

Would have to check. 

Problematic thing with this factor is we cannot directly change events count...

File: docs/test/replication-controller-details.md (hunk 4 above)
Comment (maciaszczykm) [inline docs/test/replication-controller-details.md]: Yes, but like I said before, we cannot directly change events count nor their message (warning or normal). That's why I've decided to put here factor which have impact on it and we can easily change it. To make replication controller invalid we just need to specify wrong image for example.

**Comment by maciaszczykm** [comment]
PTAL

File: docs/test/replication-controller-details.md (hunk 2 above)
Comment (maciaszczykm) [inline docs/test/replication-controller-details.md]: Done.

File: docs/test/replication-controller-details.md (hunk 1 above)
Comment (cheld) [inline docs/test/replication-controller-details.md]: ok, I see

File: docs/test/replication-controller-details.md (hunk 4 above)
Comment (cheld) [inline docs/test/replication-controller-details.md]: Yes, but on the page it is a event in the table that is marked with a warning. So, I would rather make the factor. something like 

Event warnings | image does not exist | not enough resources ...

But I would like to see more semantics than in my upper examle

**Comment by cheld** [comment]
Also add factor about heapster:

not installed | ...

How much data is shown? 24h? Can we test that? Maybe we can leave it running over night, at least

**Comment by maciaszczykm** [comment]
@cheld Applied yours suggestions.

PTAL

**Comment by cheld** [comment]
LGTM

**Pull request merged**